### PR TITLE
Combine system and clowder CA certs

### PR DESCRIPTION
**Pull request by dkuc**
I apologize for the hacky nature of this code. I tested the PR image on the cluster and it does work. To those reviewing: Feel free to modify this PR in anyway you see fit.

### CODE REVIEW

**Comment by syncrou** [comment]
@skateman I realize this fix probably isn't what you're looking for.  We were able to verify it worked ( from the PR Check image ) and test it in boundary.  Can we get this merged and then get it fixed "the right way" on a follow up?  

This code allows us to run our confidence check on the app which we were blocked on.

**Comment by marleystipich2** [comment]
This change looks good to me! 
It looks like the PR check is failing with this error:
`ERROR: deploy failed: Could not do 'wait for all deployed resources to be ready' at /var/lib/jenkins/workspace/RedHatInsights-compliance-backend-pr-check/.bonfire_venv/lib64/python3.6/site-packages/bonfire/openshift.py:79 in time`

I don't think this error is related to the changes in this PR though.

**Comment by skateman** [comment]
> This change looks good to me! It looks like the PR check is failing with this error: `ERROR: deploy failed: Could not do 'wait for all deployed resources to be ready' at /var/lib/jenkins/workspace/RedHatInsights-compliance-backend-pr-check/.bonfire_venv/lib64/python3.6/site-packages/bonfire/openshift.py:79 in time`
> 
> I don't think this error is related to the changes in this PR though. Seems like the issue is related to this recent slack convo: https://redhat-internal.slack.com/archives/CCRND57FW/p1679648639367479 Sent a message to see if we can get the pipeline fixed.

Please before saying that the pipeline issue is irrelevant, make sure that you read the k8s logs under the [build artifacts](https://ci.int.devshift.net/job/RedHatInsights-compliance-backend-pr-check/3350/artifact/artifacts/k8s_artifacts/ephemeral-wwem8x/logs/) after the tests run or follow the ephemeral namespace when it is getting deployed. Based on these it is clear that the pipeline issue IS relevant.

```
rake aborted!
SafeDownloader::DownloadError: download failed (http://compliance-ssg-service.ephemeral-wwem8x.svc:10000/datastreams/rhel8/scap-security-guide-0.1.54-5.el8/ssg-rhel8-ds.xml): Failed to open TCP connection to compliance-ssg-service.ephemeral-wwem8x.svc:10000 (execution expired)
/opt/app-root/src/app/services/safe_downloader.rb:37:in `rescue in download'
/opt/app-root/src/app/services/safe_downloader.rb:27:in `download'
/opt/app-root/src/app/services/datastream_downloader.rb:48:in `download'
/opt/app-root/src/app/services/datastream_downloader.rb:20:in `block (2 levels) in download_datastreams'
/opt/app-root/src/app/services/datastream_downloader.rb:18:in `each'
/opt/app-root/src/app/services/datastream_downloader.rb:18:in `block in download_datastreams'
/usr/share/ruby/tmpdir.rb:96:in `mktmpdir'
```

I have no problem with merging this AFTER the pipeline issue has been fixed.

File: config/initializers/0_clowder-config.rb
Comment (vkrizan) [inline config/initializers/0_clowder-config.rb:14]: [CWE-377 Insecure Temporary File](https://cwe.mitre.org/data/definitions/377.html)

Comment (dkuc) [reply config/initializers/0_clowder-config.rb:14]: @vkrizan This will be running in a container.  CWE-377 should not apply.

Comment (skateman) [reply config/initializers/0_clowder-config.rb:14]: After this is merged, someone please create a cleanup of this thing to pure ruby.

**Review by sthirugn — APPROVED** [review]
I am going to approve this, we will monitor stage tests to make sure we are good.  We will create a followup ticket to clean this code.

**Pull request merged**
